### ISSUE TITLE
Maximum call stack size exceeded on node.js 6.x

### ISSUE THREAD

**Issue by megastef** [sematext/logsene-cli#3]
I think elasticsearch client dependencies need an update: 

```

logsene search 
events.js:436
EventEmitter.listenerCount = function(emitter, type) {
                                     ^

RangeError: Maximum call stack size exceeded
    at Function.EventEmitter.listenerCount (events.js:436:38)
    at Log.listenerCount (/usr/local/lib/node_modules/logsene-cli/node_modules/elasticsearch/src/lib/log.js:68:25)
    at Function.EventEmitter.listenerCount (events.js:438:20)
    at Log.listenerCount (/usr/local/lib/node_modules/logsene-cli/node_modules/elasticsearch/src/lib/log.js:68:25)
    at Function.EventEmitter.listenerCount (events.js:438:20)
    at Log.listenerCount (/usr/local/lib/node_modules/logsene-cli/node_modules/elasticsearch/src/lib/log.js:68:25)
    at Function.EventEmitter.listenerCount (events.js:438:20)
    at Log.listenerCount (/usr/local/lib/node_modules/logsene-cli/node_modules/elasticsearch/src/lib/log.js:68:25)
    at Function.EventEmitter.listenerCount (events.js:438:20)
    at Log.listenerCount (/usr/local/lib/node_modules/logsene-cli/node_modules/elasticsearch/src/lib/log.js:68:25)
    at Function.EventEmitter.listenerCount (events.js:438:20)
    at Log.listenerCount (/usr/local/lib/node_modules/logsene-cli/node_modules/elasticsearch/src/lib/log.js:68:25)
    at Function.EventEmitter.listenerCount (events.js:438:20)
    at Log.listenerCount (/usr/local/lib/node_modules/logsene-cli/node_modules/elasticsearch/src/lib/log.js:68:25)
    at Function.EventEmitter.listenerCount (events.js:438:20)
    at Log.listenerCount (/usr/local/lib/node_modules/logsene-cli/node_modules/elasticsearch/src/lib/log.js:68:25)
```

**Comment by mbonaci** [comment]
Thanks @megastef 👍 , I'll take a look.

**Comment by megastef** [comment]
I could not reproduce the problem on another machine with the forked repository. But installing from npm caused the error above. So npm publish should help.

**Comment by megastef** [comment]
Fixed with new npm release logsene-cli@1.4.1 https://www.npmjs.com/package/logsene-cli

**Comment by mbonaci** [comment]
Thanks
